script/slim.h

```diff
@@ -21,47 +21,55 @@ TFile*f_ele=TFile::Open("../../data/fr_data_ele_"+era+".root");
 TH2D*h_m=(TH2D*)f_mu->Get("fakerate");
 TH2D*h_e=(TH2D*)f_ele->Get("fakerate");
 
-float fake_weight(int ttc_region, bool ttc_1P1F, bool ttc_0P2F, bool ttc_lep1_faketag, float l1_pt, float l1_eta, float l2_pt, float l2_eta){
+float fake_weight(int ttc_region, bool ttc_1P1F, bool ttc_0P2F, bool ttc_lep1_faketag, float l1_pt, float l1_eta, float l2_pt, float l2_eta, int iw){
   if (l1_pt>70.) l1_pt=70.;
```

Comment:
Actually a generic question I had before now I remember,

Why in Non-prompt lepton estimation for BDT there is 70 GeV restriction for l1(2)_pt and 2.3 for abs(l1(2)_eta)
https://github.com/ExtraYukawa/Script_ForMVA/blob/d2b3897698d1b610fd2e8d111450beffe7c70b77/script/slim_fake.h#L25-L28
and the actual SR has no such restrictions.
https://github.com/ExtraYukawa/Plots_code/blob/5a997e8fafe88f74eb535931686569067912b61d/TTC_DataDriven/TTC.h#L191-L197

Reply:
It's just for the boundary issue of TH2 SF.